src/System.Collections/tests/System.Collections.Tests.csproj

```diff
@@ -25,7 +25,7 @@
       <Targets>Build;DebugSymbolsProjectOutputGroup</Targets>
       <OSGroup>$(InputOSGroup)</OSGroup>
     </ProjectReference>
-    <ProjectReference Include="..\..\System.Runtime\pkg\System.Runtime.pkgproj">
+    <ProjectReference Include="..\..\System.Runtime\src\System.Runtime.csproj">
```

Comment:
We need to do this because the src/System.Collections.csproj references the live csproj as well, which means that System.Collections.dll now depends on 4.2.0.0 version of System.Runtime.dll. Since we use to have a pkgproj reference in the tests and we are testing against netcoreapp1.0, the resolving System.Runtime for the tests was the one for netstandard1.5 (version 4.1.0.0) which is why the tests were failing. We will be able to remove this once System.Runtime package version is rev'ed and we update the corefx prerelease dependencies.